### DISCUSSION TITLE
Herramientas para elección de colores en diseño

### DISCUSSION THREAD

**Discussion by DaviiidSG** [Launch-X-Latam/MisionFrontEnd#63] [Show and tell]
- ### [Picular](https://picular.co/)
Es una web en la que puedes ingresar una palabra y su IA te da colores asociados a esa palabra.

- ### [Image Color Picker](https://imagecolorpicker.com/)
Esta web te permite subir cualquier imagen y obtener su paleta de colores.

- ### [Happy Hues](https://www.happyhues.co/)
Esta web simula una landing completa que cambia según la paleta de colores que elijas para probar como quedaría una web real con esos colores.

Espero les sea útil en el futuro con sus diseños

**Comment by Ness-GM** [comment]
Oh...Muy buen dato y páginas, un abrazo compañero :3.

**Comment by dllg-lx** [comment]
A mí me gusta utilizar Coolors: https://coolors.co/

Para generar paletas visualmente agradables y que mantenga la armonía entre todos los colores.

**Comment by ryu-ed** [comment]
A mi me gusta de manera personal las funcionalidades de Adobe Color, acá les dejo el enlace.
[AdobeColor](https://color.adobe.com/es/create/color-wheel)

**Comment by renatocoello** [comment]
Tambien pueden usar [https://colorhunt.co/](url) que presenta paletas de colores completa

**Comment by OscarHub90** [comment]
Gracias! que buenas recomendaciones, muy útiles.....

**Comment by Nando-Irvin** [comment]
Muchas gracias por la recomendaciones
